recipes/r-phangorn/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = '2.2.0' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-phangorn
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: phangorn_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/phangorn_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/phangorn/phangorn_{{ version }}.tar.gz
+  sha256: 32c81a1af524284c57408a5182dd3b6fd99652ae00572e4c5c05ecd08a59210b
+
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+requirements:
+  build:
+    - r-base
+    - r-matrix
+    - r-rcpp
+    - r-ape >=4.1
+    - r-fastmatch
+    - r-igraph >=1.0
+    - r-magrittr
+    - r-quadprog
+    - gcc  # [not win]
```

Comment:
Can you please remove the "not win". We are not compiling against windows in bioconda. Also can you use llvm on OSX.